### DISCUSSION TITLE
Do we have any kind of encryption on Database to protect against external exposure?

### DISCUSSION THREAD

**Discussion by Kalyan1989** [reportportal/reportportal#2059] [General]
Do we have any kind of encryption on Database to protect against external exposure?

**Comment by DzmitryHumianiuk** [comment]
@Kalyan1989 Hi, practically ReportPortal data is encrypted in transit and at rest.

For our **on-premises** installations using PostgreSQL, encryption will depend on the specific configuration and security measures implemented within your environment. PostgreSQL does support various encryption methods to secure data, and we recommend consulting with your database administrator or IT team to ensure appropriate protections are in place.

For our cloud-based setups utilizing Amazon RDS, data encryption is available both at rest and in transit. Amazon RDS offers several built-in features to help you meet compliance and security requirements.
